### DISCUSSION TITLE
config for filter input debounce interval

### DISCUSSION THREAD

**Discussion by Janekk** [KevinVandy/material-react-table#27] [Q&A]
Is there a way to pass a custom debounce interval for filter inputs?

**Comment by KevinVandy** [comment]
Not currently. Do you think it is way off what it should be? Not sure I want to make a prop just for that.

**Reply by rajenatidan** [reply]
When using remote data, the column filter is hitting the api at every keystroke. Is there a way to configure the debounce interval before the filter is executed?

**Comment by GersonDias** [comment]
@KevinVandy do you mind if I submit a PR for that? In my case I want to configure that for the global filter text field. I can work to have a globalFilterDebounceMs prop and be sure that the current behavior still works if you don't provide that prop (500ms debounce if manualFiltering is true, otherwise 250ms).

**Comment by so-uvik** [comment]
is there now a way to set it to something custom when manualFiltering is true?
